### DISCUSSION TITLE
Create white/green papers process - Call for help!

### DISCUSSION THREAD

**Discussion by todaywasawesome** [open-gitops/project#36] [Ideas]
We've been in discussion for some time about this but there are lots of topics to explore with GitOps. These could be explored as potentially as blogposts but blogposts express more of a point of view than perhaps what we're looking for here. The idea is to create a space for the community to put forth ideas for how they implement GitOps. Topics like security, how to handle breaking glass, what happens if git is down, etc. These are all great topics! 

In this issue, we're looking for someone to take the lead on this project. Create the process, publish guidelines, and update the website so that people can easily find and submit these. Should they be called white papers? Should they be called green papers as @lloydchang suggests? I don't know, see discussion in PR [#45](https://github.com/open-gitops/website/pull/45). It would be great for someone to take and run this process.

**Comment by nishadmehendale** [comment]
I am new to this but happy to collaborate on this.

**Comment by moshloop** [comment]
I have created a WIP document here: https://github.com/open-gitops/documents/pull/49

**Comment by scottrigby** [comment]
Linking https://github.com/open-gitops/documents/pull/40 as this has come up again recently
